kpet/data.py

```diff
@@ -391,16 +391,33 @@ def __init__(self, data):
         """
         Initialize a host type.
         """
+        # TODO Drop the old schema once kpet-db is switched to the new one
+        def inherit(data):
+            if "tasks" in data:
+                data["preboot_tasks"] = data.pop("tasks")
+            return data
```

Comment:
I guess you know exactly when this will be called and it's not a problem, but I always twitch a bit when a function modifies its parameters 😅 Pity that Python doesn't have a good way to do that immutably 🙁

Reply:
Yeah, I need to clarify that in the documentation, but we can change the data. Yeah, I also feel this is missing in Python, but being slack about such things is what makes it good for fast prototyping.